physics/pace/physics/physics_state.py

```diff
@@ -212,36 +319,55 @@ def __post_init__(
     def init_zeros(cls, quantity_factory, active_packages: List[str]) -> "PhysicsState":
         initial_storages = {}
         for _field in fields(cls):
-            if len(_field.type.axes) == 2:
-                dims = [pace.util.X_DIM, pace.util.Y_DIM]
-            else:
-                dims = [pace.util.X_DIM, pace.util.Y_DIM, pace.util.Z_DIM]
-            initial_storages[_field.name] = quantity_factory.zeros(
-                dims,
-                _field.metadata["units"],
-                dtype=float,
-            ).storage
+            if "dims" in _field.metadata.keys():
+                initial_storages[_field.name] = quantity_factory.zeros(
+                    _field.metadata["dims"], _field.metadata["units"], dtype=float
+                ).storage
         return cls(
             **initial_storages,
             quantity_factory=quantity_factory,
             active_packages=active_packages,
         )
 
+    @classmethod
+    def init_from_storages(
+        cls,
+        storages: Mapping[str, Any],
+        sizer: pace.util.GridSizer,
+        quantity_factory: pace.util.QuantityFactory,
+        active_packages: List[str],
+    ):
+        inputs = {}
```

Comment:
```suggestion
        inputs: Dict[str, pace.util.Quantity] = {}
```
Will also need to import `Dict` from typing at the top of the module.